scaling_performance/cluster_limits.adoc

```diff
@@ -100,3 +100,102 @@ If you increase the number of nodes to 20, then the pod distribution changes to
 ----
 2200 / 20 = 110
 ----
+
+[[scaling-performance-planning-your-environment-according-to-application-requirements]]
+== Planning Your Environment According to Application Requirements
+
+You might have a deployment such as {product-title} running on virtual machines
+(VMs) on top of Red Hat OpenStack Platform with application requirements such
```

Comment:
Because we cannot quota or QoS on network, I'd like you to drop the Network Bandwidth column to avoid confusion.